### DISCUSSION TITLE
How to import Media Chrome to use it in Angular 15

### DISCUSSION THREAD

**Discussion by Sumant-S** [muxinc/media-chrome#740] [Q&A] [answered]
I am trying to use the elements of media chrome and use it in angular can anyone please suggest how to import the components of media chrome in angular 15,

I tried to import the elements by adding import in app.module,
```
import mediachrome from 'media-chrome';

@NgModule({
  declarations: [
    AppComponent,
    MyCustomElementComponent
  ],
  imports: [
    BrowserModule,
    AppRoutingModule,
    mediachrome
],
  providers: [],
  bootstrap: [AppComponent]
})
```
I also tried to import individual components using,
```
import {MediaController} from 'media-chrome' 

@NgModule({
  declarations: [
    AppComponent,
    MyCustomElementComponent
  ],
  imports: [
    BrowserModule,
    AppRoutingModule,
    MediaController
],
  providers: [],
  bootstrap: [AppComponent]
})

```
These two methods are not working, how can you import and use it in angular 15

**Comment by luwes** [comment, marked as answer]
here you go https://codesandbox.io/s/media-chrome-angular-43tlgh
no need for JS references. Media Chrome uses custom HTML elements which can be used as regular HTML once registered
